lib/dialects/sqlite3/schema/ddl.js

```diff
@@ -328,7 +328,7 @@ assign(SQLite3_DDL.prototype, {
         }
 
         const updatedDefs = defs
-          .split(',')
+          .split(/,(?![^(]*\))/g)
```

Comment:
This regex is too complex for my brain :D. Could you extract it into a public constant and add a couple of unit tests that would document what it does?